tests/test_docker_splunk.py

```diff
@@ -229,6 +229,32 @@ def check_splunkd(self, username, password, name=None, scheme="https"):
             assert status == 200
         return True
 
+    def check_dmc(self, containers):
+        for container in containers:
+            container_name = container["Names"][0].strip("/")
+            splunkd_port = self.client.port(container["Id"], 8089)[0]["HostPort"]
+            if container_name == "dmc":
+                # check 1: curl -k https://localhost:8089/servicesNS/nobody/splunk_monitoring_console/configs/conf-splunk_monitoring_console_assets/settings?output_mode=json -u admin:helloworld
+                status, content = self.handle_request_retry("GET", "https://localhost:{}/servicesNS/nobody/splunk_monitoring_console/configs/conf-splunk_monitoring_console_assets/settings?output_mode=json".format(splunkd_port), 
+                                                            {"auth": ("admin", self.password), "verify": False})
+                assert status == 200
+                output = json.loads(content)
+                assert output["entry"][0]["content"]["disabled"] == False
+                # check 2: curl -k https://localhost:8089/servicesNS/nobody/system/apps/local/splunk_monitoring_console?output_mode=json -u admin:helloworld
+                status, content = self.handle_request_retry("GET", "https://localhost:{}/servicesNS/nobody/system/apps/local/splunk_monitoring_console?output_mode=json".format(splunkd_port), 
+                                                            {"auth": ("admin", self.password), "verify": False})
+                assert status == 200
+                output = json.loads(content)
+                assert output["entry"][0]["content"]["disabled"] == False
+                assert output["entry"][0]["content"]["configured"] == True
+                # check 3: curl -k https://localhost:8089/services/search/distributed/peers?output_mode=json -u admin:helloworld
+                status, content = self.handle_request_retry("GET", "https://localhost:{}/services/search/distributed/peers?output_mode=json".format(splunkd_port),
+                                                            {"auth": ("admin", self.password), "verify": False})
+                assert status == 200
+                output = json.loads(content)
+                for sh in output["entry"]:
```

Comment:
Can we add another input to this function to check that the expected number of peers matches the actual reported number of peers?